configure.ac

```diff
@@ -444,18 +444,28 @@ AC_ARG_ENABLE(ncp-app,
             enable_ncp_app=${enableval}
             enable_ncp_app_mtd=${enableval}
             enable_ncp_app_ftd=${enableval}
+            enable_ncp_app_radio=${enableval}
```

Comment:
Can we consider combining the new new `--ncp-app=radio` mode with `--enable-raw-link-api`? 
What is the difference between the two options?